samples/security/psp/sidecar-psp.yaml

```diff
@@ -3,15 +3,12 @@
 apiVersion: extensions/v1beta1
 kind: PodSecurityPolicy
 metadata:
-  name: istio-sds-uds
```

Comment:
fyi, above code link is gone as part of https://github.com/istio/istio.io/pull/6644
currently seems no instructions reference how to use sample files on istio.io

Reply:
https://istio.io/docs/tasks/security/auth-sds/#increasing-security-with-pod-security-policies